client/src/Router.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { Switch, Route, BrowserRouter } from "react-router-dom";
+
+import AddFlight from "./pages/addFlight";
+import Login from "./pages/login";
+import UpdateFlight from "./pages/updateFlight";
+
+const Router = () => (
+  <BrowserRouter>
+    <Switch>
+      <Route exact path="/" render={() => <Login />} />
```

Comment:
What do you think about using the "/" path for the customer homepage, and putting all of the admin-related pages under "/admin/"?  So then admin login is at the path `/admin/`, AddFlight is `/admin/addflight`, and so on.

I imagine that most of the visitors to the site would be customers, so this might make the site easier to use (and remember) for them.

Reply:
Done. We were worried about getting react-router-dom working that we didn't think about his :smile:

Reply:
👍haha